src/sharedHelpers/cvModel.js

```diff
@@ -0,0 +1,81 @@
+// @flow
+import { Platform } from "react-native";
+import RNFS from "react-native-fs";
+
+const modelFiles = {
+  IOSMODEL: "optimized_model.mlmodelc",
+  IOSTAXONOMY: "taxonomy.json",
+  ANDROIDMODEL: "optimized_model.tflite",
+  ANDROIDTAXONOMY: "taxonomy.csv"
+};
```

Comment:
IMO, these values should be configurable in .env. Assuming these files have unique names each time we change them, that will help us know what version of the model we're building with.

Reply:
Have refactored everything so that the filenames are read from the .env file. Please, let me know if the information in the readme would be enough to get started without any prior knowledge of how to set up the repository.